### PR TITLE
Fix/70/synchronization problem

### PR DESCRIPTION
**Pull request by blazejkrzak**
https://github.com/lukso-network/pandora-execution-engine/issues/70

This changes is a fix for synchronization problem

### CODE REVIEW

**Comment by blazejkrzak** [comment]
SH overview 
Bootnodes and ips are dynamic so genesis may change
127.0.0.1 is port forward from kubectl for particular node.
They must share the same genesis.json file and genesis.ssz



# How to run 
```sh
wget https://storage.googleapis.com/l16-common/pandora/pandora_genesis3.json -O ./genesis.json
./geth --datadir ./debug init ./genesis.json



./geth --datadir ./debug  --bootnodes enode://e9353d37e664ce0070db7b6e31d183b463ccc24d51623c8dfbf40b7abdaa9b483f134c291f0c2521ea4b2c6f7954b5359bea0c6b069e1bb62fd49b996827e396@127.0.0.1:30403 --networkid 4004181 --rpcaddr 0.0.0.0 --rpcapi admin,eth,net --rpc --rpccorsdomain "*" --miner.etherbase 0xb46d14ef42ac9bb01303ba1842ea784e2460c7e5 --verbosity 10 --miner.notify "ws://34.141.196.72:7878,http://34.141.196.72:7877" -port 30308 -rpcport 8530
```

Those ips may vary,
Those are ips of orchestrator node. You can run this manually
`--miner.notify  ws://34.141.196.72:7878,http://34.141.196.72:7877`

**Comment by blazejkrzak** [comment]
Screenshot from 2021-07-02 10-01-40

File: trie/sync_bloom.go [hunk 1]
```diff
@@ -52,6 +52,18 @@ type SyncBloom struct {
 	pend   sync.WaitGroup
 }
 
+func (b *SyncBloom) Dump() {
```
Comment (blazejkrzak) [inline trie/sync_bloom.go:55]: Not needed at all, It was only for debug

**Review by mxmar — APPROVED** [review]

**Comment by blazejkrzak** [comment]
I close this, because https://github.com/lukso-network/pandora-execution-engine/pull/67 was having partial fix. (Only increase the cache size if needed). All the other stuff was redundant (my debug or tests)

**Pull request closed without merging**